llvm/lib/SYCL/XOCCIRDowngrader.cpp

```diff
@@ -30,6 +30,7 @@
 #include "llvm/Support/FileSystem.h"
 #include "llvm/Support/Path.h"
 #include "llvm/Support/raw_ostream.h"
+#include "llvm/../../lib/IR/LLVMContextImpl.h"
```

Comment:
This looks strange

Reply:
yeah this is a dependency we rather not have. but using this is the most reliable way to eliminate poison.

Reply:
Would `#include "../lib/IR/LLVMContextImpl.h"` work too and this is just to make clear this is related to `llvm`?

Reply:
Ah actually you have already changed it!